src/index.js

```diff
@@ -0,0 +1,72 @@
+import _, { indexOf } from 'lodash';
+import './index.css';
+import {TaskList} from './TasklistFile.js';
+import {Interaction} from './interactive_List.js'
+const textInputAdd = document.getElementById('addTT');
+const taskList = document.getElementById('task-list');
+function check(status) {
+  if (status === true) { return "checked";
+ } else { return "unchecked";
+ }}
+function styleTask(status) { if (status === true) { 
+  return "task taskStyle" } else { return "task" 
+  }  }
+function display() {
+  TaskList.taskArray = JSON.parse(localStorage.getItem('coward')) || [];
+  taskList.innerHTML = ''
+ TaskList.taskArray.forEach((task, i) => {
+ taskList.innerHTML += `<div class="taskR"><input type="checkbox" ${check(TaskList.taskArray[i].completed)} 
+ class="box" value=${i}> <p id=${i} class="${styleTask(TaskList.taskArray[i].completed)}" title="Double Click to
+  Edit & Enter to Save">${TaskList.taskArray[i].description}</p><div class="img" id="deleteBtn ${i}" title="${i}"></div></div>`;
+  }  );
+}
+display();
+//Add Task Event Listner
+document.querySelector('.save').addEventListener('click', (e) => {
+   TaskList.addTask(textInputAdd.value);
+    location.reload();
+})
+//Add Task Event Listner
+textInputAdd.addEventListener('keypress',  (e) => {
+    if (e.key === 'Enter') { TaskList.addTask(textInputAdd.value);
+   location.reload();}
+ })
+ //Delete Task Event Listner
+  const deleteBtn = document.querySelectorAll('.img');
+   deleteBtn.forEach((Btn) => {
+          Btn.addEventListener('click', (e) => {
+                TaskList.delete(Btn.title);
+         location.reload();
+    });
+  });
+//EditTask Event Listner
+const taskArray = document.querySelectorAll('.task');
+taskArray.forEach((task) => {
+  task.addEventListener('dblclick', (e) => {
```

Comment:
In this line, you introduced the parameter e for the function passed inside the event listener, but it is never used, so please remove this parameter.